### DISCUSSION TITLE
Hey, did I miss something

### DISCUSSION THREAD

**Discussion by perlchild** [dashnine/PaperBlossoms#240] [General]
Is there a way for my character to start a second title?
I've completed the Emerald Magistrate title(30xp, and my paperblossom does count out an exact 30xp), and my ST would like us to earn at least another to pace our advancement.
I've checked the book, it's clearly the intent that once you've completed a title, you can start another, but I don't see the option in paper blossom.
Any thoughts?

**Comment by dashnine** [comment]
Howdy!  I took a look -- in the latest ( https://github.com/dashnine/PaperBlossoms/releases/tag/v1.4.0-beta ) at the very least, it still functions -- once you've spent enough XP to complete the title, it adds the title's ability to your character sheet, clears the title, and re-enables the 'Add' button to start a different title. 

If you've confirmed you have enough XP, maybe confirm that when adding the title the amount matches the expected XP in the book?  It could always be a data issue with a specific title. :)
